packages/smart-vaults/src/vault/SmartVault.sol

```diff
@@ -37,11 +37,22 @@ contract SmartVault is IAccount, Ownable, UUPSUpgradeable, MultiSignerAuth, ERC1
 
     /// @notice Single User Op Signature Scheme.
     struct SingleUserOpSignature {
+        /// @notice Upper limit for Max Priority Fee Per Gas that should be charged by the userOp. This is included in
+        /// the light userOp hash to ensure last signer does not exceed the specified gas price. Can be `0` if
+        /// threshold is 1.
```

Comment:
"Ignored if set to `0`" or something at the end instead of your current last line i think?